core/src/main/scala/org/apache/spark/Partitioner.scala

```diff
@@ -21,6 +21,8 @@ import java.io.{IOException, ObjectInputStream, ObjectOutputStream}
 
 import scala.collection.mutable
 import scala.collection.mutable.ArrayBuffer
+import scala.language.existentials
```

Comment:
Curious, why was this required ?

Reply:
Without this import, there was a compiler warning:
```
Warning:(63, 29) inferred existential type Option[org.apache.spark.rdd.RDD[_$2]]( forSome { type _$2 }), which cannot be expressed by wildcards,  should be enabled
by making the implicit value scala.language.existentials visible.
This can be achieved by adding the import clause 'import scala.language.existentials'
or by setting the compiler option -language:existentials.
See the Scaladoc for value scala.language.existentials for a discussion
why the feature should be explicitly enabled.
```

Spark build failed because of this.

Reply:
If we explicitly set the type, is it still required ? For example, with `val hasMaxPartitioner: Option[RDD[_]] = ...` ?